### DISCUSSION TITLE
Issue running cert-gen script

### DISCUSSION THREAD

**Discussion by tuptaker** [dexidp/dex#2109] [Q&A]
After running cert-gen, it appears the script is hanging during one of the openssl genra commands. What am I missing here? I don't see that the ca.crt, server.crt, or client.crt are generated, and the bash script never terminates. Execution and output below:

export SAN=IP.1:127.0.0.1
 ./cert-gen
 Creating CA, server cert/key, and client cert/key...
Generating RSA private key, 4096 bit long modulus (2 primes)
............................................++++
...........++++
e is 65537 (0x010001)

**Comment by nabokihms** [comment]
It is hard to tell what goes wrong. Maybe it is better to rewrite the script using something more environment-independent like docker.

I will check this. Thanks for reporting.
